src/com/facebook/buck/android/FilterResourcesSteps.java

```diff
@@ -457,6 +466,7 @@ public static Builder builder() {
     @Nullable private ProjectFilesystem filesystem;
     @Nullable private ImmutableBiMap<Path, Path> inResDirToOutResDirMap;
     @Nullable private ResourceFilter resourceFilter;
+    private Optional<String> localizedStringFileName;
```

Comment:
This is not initialized, it should probably be set to Optional.empty()

Reply:
good catch, fixed.